pkg/kubelet/stats/stats_provider_test.go

```diff
@@ -59,6 +59,8 @@ const (
 	offsetFsTotalUsageBytes
 	offsetFsBaseUsageBytes
 	offsetFsInodeUsage
+
+	cbr0NetworkSeed = 100
```

Comment:
nit: call this something other than seed, since it's not seeding anything. Maybe value, since it's fixed?

Reply:
Will do

Reply:
Done. PTAL.